pkgs/development/compilers/icedtea-web/default.nix

```diff
@@ -1,17 +1,31 @@
-{ stdenv, fetchurl, jdk, gtk2, xulrunner, zip, pkgconfig, perl, npapi_sdk, bash, bc }:
+{ stdenv, fetchurl, jdk, glib, xulrunner, zip, pkgconfig, perl, npapi_sdk, bash, bc, git }:
+let
+
+cveFixes = fetchurl {
```

Comment:
Can you put something similar to what's in your summary here?
i.e what cve's are fixed?

Reply:
done